.github/workflows/gh-pages.yml

```diff
@@ -10,11 +10,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-node@v3
-      with:
-        node-version: 16
```

Comment:
Is there a reason to not specify the Node version?

Reply:
Well, we did not need to use `16`. Funnily enough, it appears that `16` is the current default 😆 

Maybe it would be good to actually start using `node-version: latest` or `node-version: lts`?

Reply:
`lts` would be a better option